src/nci/NfcTagManager.cpp

```diff
@@ -250,6 +253,10 @@ int NfcTagManager::reconnectWithStatus()
     retCode = reSelect(NFA_INTERFACE_ISO_DEP);
   } else if (tag.mTechLibNfcTypes[0] == NFA_PROTOCOL_T2T) {
     retCode = reSelect(NFA_INTERFACE_FRAME);
+  } else if (isMifareTech(tag.mTechLibNfcTypes[0])) {
+#ifdef NFCC_PN547
```

Comment:
I saw libnfc-nci has already defined NFA_INTERFACE_MIFARE, is this #ifdef needed?